code/modules/mining/mine_turfs.dm

```diff
@@ -36,9 +37,43 @@
 	var/obj/item/last_find
 	var/datum/artifact_find/artifact_find
 	var/ignore_mapgen
-
+	var/ignore_oregen = FALSE
```

Comment:
double negated variable, it would be better to do
var/do_oregen and var/do_cavegen

Reply:
It would be nicer to do that but that'd require me to touch cavegen/mapgen/oregen and that's beyond the scope of this pr. The reason the var looks new is that I saw an opportunity to get rid of a _vr file which only had those var definitions in them